server_environment/readme/ROADMAP.md

```diff
@@ -1,5 +1,5 @@
 - it is not possible to set the environment from the command line. A
-  configuration file must be used.
```

Comment:
these changes to server_env are not related. If you want to keep them here, pls move them to their own commit